internal/config/job.go

```diff
@@ -13,13 +14,36 @@ type JobSubmit struct {
 
 	// Flags
 	Filename     string
+	Attach       string
+	Timeout      time.Duration
 	TemplateVals TemplateVals
 }
 
 func (c *JobSubmit) AddFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVarP(&c.Filename, "filename", "f", "", "YAML or JSON file containing the jobs creation request")
 	cmd.MarkFlagRequired("filename")
 	cmd.Flags().Var(&c.TemplateVals, "set", "--set var=val")
+
+	cmd.Flags().StringVarP(&c.Attach, "attach", "", "", "Waits until the job runs exits. Accept stdout or stderr as param")
+
+	cmd.Flags().DurationVar(&c.Timeout, "timeout", 0, "Timeout when waiting any retry or connection fails. If timeout is 0, will wait for completion or cancellation")
+
+	cmd.Flags().Lookup("attach").NoOptDefVal = "stdout"
+}
+
+func (c *JobSubmit) ValidateAttachFlag(cmd *cobra.Command) bool {
+	attach := cmd.Flags().Lookup("attach")
+
+	if !attach.Changed {
+		return true
+	}
+
+	switch attach.Value.String() {
+	case "stdout", "stderr":
+		return true
+	}
+
+	return false
```

Comment:
I'm not sure this is working as expected, e.g.:

```
$ go run ./cmd/signadot/ job submit -f /home/ddv/git/signadot/signadot/sandboxes/test/control/data/jobs/basic.yaml --attach stdout
Error: unknown command "stdout" for "signadot job submit"
exit status 1

```

Reply:
$ go run ./cmd/signadot/ job submit --attach=stdout


That's the proper usage

Reply:
Oh, that's strange, why the other flags works without the "=" and not this one?.
E.g.:

```bash
$ go run ./cmd/signadot/ job get basic-0xgdwza --output yaml
createdAt: 2024-06-14T12:11:41Z
name: basic-0xgdwza
spec:
...
```

```bash
$ go run ./cmd/signadot/ job get basic-0xgdwza --output=yaml
createdAt: 2024-06-14T12:11:41Z
name: basic-0xgdwza
spec:
...
```

Reply:
I think it is because `--attach` has an optional argument but `--output` does not

Reply:
Correct, this is the reference to the docs
https://github.com/spf13/pflag?tab=readme-ov-file#command-line-flag-syntax